gatsby-config.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  plugins: [
+    {
+      resolve: `gatsby-plugin-prefetch-google-fonts`,
```

Comment:
nice plugin, I will use it as well :D 
But I have some lacks here:
`gatsby-plugin-styled-components`
there is no translation set up (you can check: `gatsby-plugin-intl`),
`gatsby-plugin-eslint` <- this will help you with eslint.

Maybe there are plans for those plugins, then 🆗 do it when you plan

Reply:
gatsby-plugin-styled-components is already in place, other two indeed were in future plans but to be sure I added them as a TODO comment